src/More/About.tsx

```diff
@@ -8,23 +8,29 @@ import {
   Text,
   View,
 } from "react-native"
+import {
+  getApplicationName,
+  getBuildNumber,
+  getVersion,
+} from "react-native-device-info"
 
-import packageJson from "../../package.json"
 import { RTLEnabledText } from "../components/RTLEnabledText"
 
 import { Colors, Spacing, Typography } from "../styles"
 
 export const AboutScreen: FunctionComponent = () => {
   const { t } = useTranslation()
 
+  const versionInfo = `${getVersion()} ${getBuildNumber()}`
```

Comment:
Very arbitrary decision here, not sure if we want to add a `B` in the middle.

Reply:
maybe `version: 0.0.1 (18)`. as this is the same format that apple uses?

Reply:
yeah actually, we likely want to do something like this as `0.0.1 1` could easily be mistaken for `0.0.11`